include/common.h

```diff
@@ -590,7 +599,9 @@ struct DrawInfo {
     DISPENV dispenv;
     DRAWENV drawenv; // 0x14
     u32 unk70;
-    u8 pad2[0x25];
+    u8 pad70[10];
```

Comment:
Nit: should be 0x80 - 0x74 = 12 bytes of padding no? Or are you just adding enough to get the next word-aligned field to be at the right location?

Reply:
Ok I increased the padding